.github/workflows/jest.yml

```
@@ -0,0 +1,32 @@
+# Runs jest based unit tests for frontend javascript and @jupyterhub/binderhub-client
+name: "JS Unit tests"
+
+on:
+  pull_request:
+    paths: &paths
```

This workflow hasn't been run in this PR yet.
```suggestion
    paths: &paths
      - .github/workflows/jest.yml
```

---

Hmm, I added this but it doesn't seem to be running still. It was running before (https://github.com/jupyterhub/binderhub/actions/runs/6387299049/job/17335421212)

---

Ah because github actions doesn't support YAML anchors https://github.com/actions/runner/issues/1182